docs/pipelineruns.md

```diff
@@ -844,6 +847,518 @@ For more information, see the following topics:
 [`Custom tasks`](pipelines.md#using-custom-tasks) may or may not use workspaces.
 Consult the documentation of the custom task that you are using to determine whether it supports workspaces.
 
+#### Propagated Workspaces
+
+**([alpha only](https://github.com/tektoncd/pipeline/blob/main/docs/install.md#alpha-features))**
```

Comment:
could we also list this feature in https://github.com/tektoncd/pipeline/blob/main/docs/install.md#alpha-features?

Reply:
sure